src/client/js/Controls/PropertyGrid/Widgets/BlobHashWidget.js

```diff
@@ -0,0 +1,174 @@
+/*globals define, $, WebGMEGlobal*/
+/*jshint browser: true*/
+/**
+ * @author kecso / https://github.com/kecso
+ */
+
+define([
+    'js/Controls/PropertyGrid/Widgets/WidgetBase',
+    'blob/BlobClient',
+    'js/logger',
+    'common/regexp',
+    'css!./styles/BlobHashWidget.css'
+], function (WidgetBase,
```

Comment:
It would have been a bit more elegant if this could share some implementation with the AssetWidget instead of copying the common code.